code/modules/mob/living/carbon/carbon.dm

```diff
@@ -273,6 +273,25 @@
 
 	if(!item) return
 
+	if(istype(item, /obj/item/stack/throwing_knife))
+		var/obj/item/stack/throwing_knife/V = item
+		var/ROB_throwing_damage = src.stats.getStat(STAT_ROB)
+		V.throwforce = (35/(1+100/(ROB_throwing_damage))+10)
```

Comment:
up to 45~ damage with diminishing returns as it approaches it?

Reply:
(check //note)
soft cap; This would result in knives doing 10 damage at 0 rob, 20 at 50 ROB, 25 at 100 etc.